components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -203,53 +206,85 @@ func runPatchCommand(cmd *cobra.Command, args []string) error {
 				writer.Printf(output + "\n")
 				writer.Success("Patching is completed on " + remoteService + " node : " + frontendIps[i] + "\n")
 			}
+
 		}
 		if configCmdFlags.postgresql {
 			const remoteService string = "postgresql"
-			tomlFilePath, err := getMergerTOMLPath(args, infra, timestamp, remoteService, GET_CONFIG)
+			//checking for log configuration
+			err := enableCentralizedLogConfigForHA(args, remoteService, sshUser, sshPort, sskKeyFile, infra.Outputs.PostgresqlPrivateIps.Value)
 			if err != nil {
 				return err
 			}
-			scriptCommands := fmt.Sprintf(BACKEND_COMMAND, dateFormat, remoteService, "%s", remoteService+timestamp)
-			if len(infra.Outputs.PostgresqlPrivateIps.Value) > 0 {
-				remoteIp := infra.Outputs.PostgresqlPrivateIps.Value[0]
-				err := copyFileToRemote(sskKeyFile, tomlFilePath, sshUser, remoteIp, remoteService+timestamp, true)
+			//checking database configuration
+			existConfig, reqConfig, err := getExistingAndRequestedConfigForPostgres(args, infra, remoteService, GET_CONFIG)
+			if err != nil {
+				return err
+			}
+			isConfigChangedDatabase := isConfigChanged(existConfig, reqConfig)
+			//Implementing the config if there is some change in the database configuration
+			if isConfigChangedDatabase {
+				tomlFile := args[0] + timestamp
+				tomlFilePath, err := createTomlFileFromConfig(&reqConfig, tomlFile)
 				if err != nil {
-					writer.Errorf("%v", err)
 					return err
 				}
-				output, err := ConnectAndExecuteCommandOnRemote(sshUser, sshPort, sskKeyFile, remoteIp, scriptCommands)
-				if err != nil {
-					writer.Errorf("%v", err)
-					return err
+				scriptCommands := fmt.Sprintf(BACKEND_COMMAND, dateFormat, remoteService, "%s", remoteService+timestamp)
+				if len(infra.Outputs.PostgresqlPrivateIps.Value) > 0 {
+					remoteIp := infra.Outputs.PostgresqlPrivateIps.Value[0]
```

Comment:
This is the code which is already there, as this is for database, hab takes care of it. If its being patched in one node it will applied to other node as well.